gocd/templates/libs/relay.libsonnet

```diff
@@ -0,0 +1,91 @@
+local pops = import './relay-pops.libsonnet';
+local gocdtasks = import 'github.com/getsentry/gocd-jsonnet/v1.0.0/gocd-tasks.libsonnet';
+
+function(region) {
+  environment_variables: {
+    SENTRY_REGION: region,
+  },
+  group: 'relay-next',
+  lock_behavior: 'unlockWhenFinished',
+  materials: {
+    relay_repo: {
+      git: 'git@github.com:getsentry/relay.git',
+      shallow_clone: true,
+      branch: 'master',
+      destination: 'relay',
+    },
+  },
+  stages: [
+
+    // Check that github status is good
+    {
+      checks: {
+        fetch_materials: true,
+        jobs: {
+          checks: {
+            environment_variables: {
+              GITHUB_TOKEN: '{{SECRET:[devinfra-github][token]}}',
+            },
+            timeout: 1800,
+            elastic_profile_id: 'relay',
+            tasks: [
+              gocdtasks.script(importstr '../bash/github-check-runs.sh'),
+            ],
+          },
+        },
+      },
+    },
+
+    // Deploy relay
+    {
+      'deploy-production': {
+        fetch_materials: true,
+        jobs: {
+          create_sentry_release: {
+            environment_variables: {
+              SENTRY_ORG: 'sentry',
+              SENTRY_PROJECT: 'relay',
+              SENTRY_URL: 'https://sentry.my.sentry.io/',
+              // Temporary; self-service encrypted secrets aren't implemented yet.
+              // This should really be rotated to an internal integration token.
+              SENTRY_AUTH_TOKEN: '{{SECRET:[devinfra-temp][relay_sentry_auth_token]}}',
+            },
+            timeout: 1200,
+            elastic_profile_id: 'relay',
+            tasks: [
+              gocdtasks.script(importstr '../bash/create-sentry-release.sh'),
+            ],
+          },
+          deploy: {
+            timeout: 1200,
+            elastic_profile_id: 'relay',
+            tasks: [
+              gocdtasks.script(importstr '../bash/deploy-relay.sh'),
+            ],
+          },
+        },
+      },
+    },
+
+    {
+      'progress-to-pops': {
+        approval: {
+          type: 'manual',
+          allow_only_on_success: true,
```

Comment:
Does a manual approval mean that someone needs to click to proceed with the deployment? My understanding is that the goal was to do this automatically.

Reply:
That is up to the folks owning the pipeline.

Currently relay-pops is a manual pipeline and this preserves that.

I think it's worth keeping for now to be as close to the current setup and then review it once everyone is more comfortable with pipedream deployments.

Reply:
Sounds good to me!